libraries/helpers.rb

```diff
@@ -1,6 +1,9 @@
 module OslPhp
   module Cookbook
     module Helpers
+      def php_version
+      end
+
```

Comment:
This needs to be removed. It's conflicting with the same method in the php cookbook.